backend/.env.example

```diff
@@ -2,6 +2,7 @@
 FLASK_APP=app.py
 FLASK_ENV=development
 SITE_URL="https://zfnd.org" # No trailing slash
+ADMIN_SITE_URL="https://grants-admin.zfnd.org" # No trailing slash
```

Comment:
Cat's out of the bag, but let's put the localhost one in the env example. The less this url is publicized, the better imo.